### PR TITLE
Return more informative error if comparison data not avaialble

### PR DESCRIPTION
**Pull request by r-ash**

### CODE REVIEW

File: tests/testthat/test-hintr-plot-data.R [hunk 1]
```diff
@@ -32,6 +32,17 @@ test_that("comparison plot returns useful error if run with old naomi output", {
...
 
+test_that("comparison plot returns useful error if no input output data", {
+  t <- tempfile()
+  output_data <- readRDS(a_hintr_output$model_output_path)
+  output_data$output_package$inputs_outputs <- NULL
+  saveRDS(output_data, t)
+  hintr_output <- a_hintr_output
+  hintr_output$model_output_path <- t
+  expect_error(hintr_comparison_plot(hintr_output),
+               "Model output out of date please re-run model and try again")
```
Comment (jeffeaton) [inline tests/testthat/test-hintr-plot-data.R:43]: Can you please add punctuation on this error message?
```
"Model output out of date. Please re-run model and try again."
```

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# naomi 2.7.6
+
+* `hintr_comparison_plot` will error if comparison plot data does not exist.
```
Comment (jeffeaton) [inline NEWS.md:3]: Edit `hints_comparison_plot` -> `hintr_comparison_plot()` for clarity.

**Review by jeffeaton — CHANGES_REQUESTED** [review]
Thanks, looks good; just a couple of small requests.

I am on leave next week so please don't wait for me to review again to merge.

**Comment by r-ash** [comment]
hintr PR https://github.com/mrc-ide/hintr/pull/406

**Review by rtesra — APPROVED** [review]

**Pull request merged**